src/VCEL.JS/JsPropertyValueAccessor.cs

```diff
@@ -47,12 +47,10 @@ public string GetValue(IContext<string> context)
                 finalPropOrMethod = jsDateMethod;
             }
 
-            if (context.Value.Contains(Constants.DefaultContext))
-                return $"{Constants.DefaultContext}.{finalPropOrMethod}";
-
-            return monad.Lift(context.Value == "{ }"
-                ? $"{Constants.DefaultContext}.{finalPropOrMethod}"
-                : $"{context.Value}.{finalPropOrMethod}");
+            var jsObjContext = context as JsObjectContext;
+            return monad.Lift(jsObjContext?.Object is string
```

Comment:
Here we check that if the context is already processed, 
if it does, then it must be a string (concatenated)
in that case we dont want to append the vcelContext again.